app/assets/javascripts/admin/components/admin-report.js.es6

```diff
@@ -41,7 +41,7 @@ export default Ember.Component.extend({
   classNameBindings: ["isEnabled", "isLoading", "dasherizedDataSourceName"],
   classNames: ["admin-report"],
   isEnabled: true,
-  disabledLabel: "admin.dashboard.disabled",
+  disabledLabel: I18n.t("admin.dashboard.disabled"),
```

Comment:
@jjaffeux I had to change `disabledLabel` from being an I18n key to the actual translated string. I hope that doesn't create any problems.